docs/src/tutorials/hybrid-applications-with-webpack/template-deep-dive/building-your-project.md

```diff
@@ -0,0 +1,23 @@
+# Building your project
+
+We are writing a package that should be used in the browser, so we run this in our terminal:
+
+```bash
+$ wasm-pack build
```

Comment:
Should this perhaps instead recommend `npm run build`? Here with webpack configured the `wasm-pack` part of things is only half the build process I think?